src/components/Nav/types.ts

```diff
@@ -9,7 +9,10 @@ type LinkXorItems = LinkOnly | ItemsOnly
 export type NavItem = {
   label: string
   description: string
-  icon?: IconType | ((props: IconProps) => JSX.Element)
+  icon?:
+    | IconType
+    | FC<SVGProps<SVGElement>>
+    | ((props: IconProps) => JSX.Element)
```

Comment:
Do we still need to account for this Chakra `((props: IconProps) => JSX.Element)` type? Or can we rm this along with the import?

Reply:
Ahh good catch, didn't notice that. Will remove it in a new PR.